torch/export/_trace.py

```diff
@@ -551,9 +551,14 @@ def __init__(self, mod):
 
                     def forward(self, *args, **kwargs):
                         nonlocal out_spec
-                        flat_outs, out_spec = pytree.tree_flatten(
-                            self._export_root(*args, **kwargs)
-                        )
+                        if isinstance(self._export_root, torch.fx.GraphModule):
+                            with torch.fx.traceback.preserve_node_meta():
+                                tree_out = torch.fx.Interpreter(self._export_root).run(
```

Comment:
@titaiwangms @suo the `torch.fx.Interpreter().run()` is used here to preserve `metadata` attribute?

Reply:
yes, plus: `with torch.fx.traceback.preserve_node_meta():`